### PR TITLE
fix(addresses-creation)

### PR DESCRIPTION
**Pull request by Noam6421**
Fix: removed update address functionality and return to always creating new address.

### CODE REVIEW

File: server/src/ClientToDBAPI/ClinicalDetailsRoute/mainRoute.ts [hunk 1]
```diff
@@ -202,42 +202,27 @@ const saveClinicalDetails = (request: Request, response: Response, baseLog: Init
...
         const createAddressLogger = logger.setup({...logData, workflow: `${logData.workflow}: create isolation address`})
-
         const parameters = {input: requestAddress};
         createAddressLogger.info(launchingDBRequestLog(parameters), Severity.LOW);
-
-        graphqlRequest(CREATE_ADDRESS, response.locals, parameters)
-        .then((result: CreateAddressResponse) => {
+        graphqlRequest(CREATE_ADDRESS, response.locals, {
+            input: requestAddress
+        }).then((result: CreateAddressResponse) => {
```
Comment (RoeiRom) [inline server/src/ClientToDBAPI/ClinicalDetailsRoute/mainRoute.ts:218]: change {....} to parameters

**Review by RoeiRom — CHANGES_REQUESTED** [review]

File: server/src/ClientToDBAPI/ClinicalDetailsRoute/mainRoute.ts [hunk 2]
```diff
@@ -202,42 +202,27 @@ const saveClinicalDetails = (request: Request, response: Response, baseLog: Init
...
         const requestAddress: InsertAndGetAddressIdInput = formatToInsertAndGetAddressIdInput(isolationAddress);
         const createAddressLogger = logger.setup({...logData, workflow: `${logData.workflow}: create isolation address`})
-
         const parameters = {input: requestAddress};
         createAddressLogger.info(launchingDBRequestLog(parameters), Severity.LOW);
-
-        graphqlRequest(CREATE_ADDRESS, response.locals, parameters)
-        .then((result: CreateAddressResponse) => {
+        graphqlRequest(CREATE_ADDRESS, response.locals, {
+            input: requestAddress
```
Comment (barnahum8) [inline server/src/ClientToDBAPI/ClinicalDetailsRoute/mainRoute.ts:217]: you can still use the 'parameters' variable

Comment (Noam6421) [reply server/src/ClientToDBAPI/ClinicalDetailsRoute/mainRoute.ts:217]: 👍🏼

File: server/src/ClientToDBAPI/ClinicalDetailsRoute/mainRoute.ts (hunk 1 above)
Comment (Noam6421) [reply server/src/ClientToDBAPI/ClinicalDetailsRoute/mainRoute.ts]: 👍🏼

**Review by barnahum8 — APPROVED** [review]

**Review by RoeiRom — APPROVED** [review]

**Pull request merged**